libnavigation-core/src/main/java/com/mapbox/navigation/core/fasterroute/FasterRouteRequestCallback.kt

```diff
@@ -0,0 +1,34 @@
+package com.mapbox.navigation.core.fasterroute
+
+import com.mapbox.api.directions.v5.models.DirectionsRoute
+import com.mapbox.api.directions.v5.models.RouteOptions
+import com.mapbox.navigation.base.route.Router
+
+interface FasterRouteRequestCallback {
```

Comment:
Could we just reuse `RoutesRequestCallback`?

Reply:
I am afraid not. 👀 at 
```
fun onRouteReady(route: DirectionsRoute): Boolean
```